js/index.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Derived from https://github.com/novnc/noVNC/blob/master/vnc_lite.html, which was licensed
```

Comment:
```suggestion
 * Derived from https://github.com/novnc/noVNC/blob/v1.4.0/vnc_lite.html, which was licensed
```
I know it's unlikely, but if the licensing on the master branch changes in future including the version will avoid confusion